Library/Homebrew/requirements/php_requirement.rb

```diff
@@ -0,0 +1,11 @@
+class PhpRequirement < Requirement
+  fatal true
+  satisfy { which "php" }
+
+  env do
+    if which("php") != "/usr/bin/php"
```

Comment:
This adds `/usr/local/bin/php-config` to the `$PATH` variable that homebrew sets up. This is necessary if you want to link this against your homebrew version of PHP, otherwise it'll use the system one.